### PR TITLE
User can buy a subscription

### PR DESCRIPTION
**Pull request by johannaanden**
## Features
- Adds stripe implementation
- Adds testing for stripe implementation
- Adds subscriber to enum as role number 1
- Adds controller actions to subscriber controller in content management
- Changes the role when the payment was successful

## Gem
- stripe-rails
- chromedriver-helper
- selenium-webdriver

## Pivotal tracker
[PT Link](https://www.pivotaltracker.com/story/show/163088248) 

## Screenshot
buy_subscription

### CODE REVIEW

File: app/controllers/content_management/articles_controller.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 class ContentManagement::ArticlesController < ApplicationController
 
+    before_action :get_article, only: [:show, :edit, :update]
+
     def index
         @articles = Article.all
     end
 
     def show
```
Comment (oliverochman) [inline app/controllers/content_management/articles_controller.rb:9]: Are you using this route for anything?

File: app/controllers/subscribers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+class SubscribersController < ApplicationController
+
+    def new
```
Comment (oliverochman) [inline app/controllers/subscribers_controller.rb:3]: why do you have the `new` route defined with no logic in it?

File: features/Journalist_can_update_article.feature [hunk 3]
```diff
@@ -8,10 +8,12 @@ Feature: Journalist can create article
         Given the following user exists
             | name | email          | password | password_confirmation | role       |
             | Bill | bill@email.com | password | password              | journalist |
+
```
Comment (oliverochman) [inline features/Journalist_can_update_article.feature:11]: Dont clutter up the PR with changes that has nothing to do with the feature

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: app/controllers/subscribers_controller.rb (hunk 2 above)
Comment (GergKllai1) [reply app/controllers/subscribers_controller.rb]: We have a before_action:
`before_action :get_article, only: [:show, :edit, :update]`

To perform this:
```
def get_article
        @article = Article.find(params[:id])
    end
```

This was to make the code more DRY

File: app/controllers/content_management/articles_controller.rb (hunk 1 above)
Comment (GergKllai1) [reply app/controllers/content_management/articles_controller.rb:9]: Yes we have a show view

File: app/controllers/subscribers_controller.rb (hunk 2 above)
Comment (GergKllai1) [reply app/controllers/subscribers_controller.rb:3]: I though we needed for the rout and view.
Removed

File: features/Journalist_can_update_article.feature (hunk 3 above)
Comment (tochman) [reply features/Journalist_can_update_article.feature:11]: I asked them to add a new line between steps in the background 😉

Comment (GergKllai1) [reply features/Journalist_can_update_article.feature:11]: This was a request from Thomas

File: app/controllers/content_management/articles_controller.rb (hunk 1 above)
Comment (GergKllai1) [reply app/controllers/content_management/articles_controller.rb:9]: And a before action to get the articles, that's why it seems empty, but it is not ;)

**Pull request merged**